ihatemoney/templates/list_bills.html

```diff
@@ -134,6 +134,9 @@ <h3 class="modal-title">{{ _('Add a bill') }}</h3>
                 <td class="bill-actions">
                     <a class="edit" href="{{ url_for(".edit_bill", bill_id=bill.id) }}" title="{{ _("edit") }}">{{ _('edit') }}</a>
                     <a class="delete" href="{{ url_for(".delete_bill", bill_id=bill.id) }}" title="{{ _("delete") }}">{{ _('delete') }}</a>
+                    {% if bill.external_link %}
+                    <a class="see" href="{{ bill.external_link }}" ref="noopener" target="_blank" title="{{ _("see") }}">{{ _('see') }} </a>
```

Comment:
Should we replace "See" by "See attached document"?

Reply:
I have no preference tell me what's the best in your opinion :)

Reply:
I think [explicit is better than implicit](https://www.python.org/dev/peps/pep-0020/) :-)